### ISSUE TITLE
Heading level should not be identified as related to "size"

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#4407]
For no reason, ever, the interface should suggest users the heading levels are related to the font "size". Headings have been abused for years because of lack of awareness and because many users used them just for their size.

Currently, in the sidebar, the levels have a "Size" label:

![screen shot 2018-01-11 at 15 19 20](https://user-images.githubusercontent.com/1682452/34830670-e1c87336-f6e4-11e7-941a-abb1700ce6e2.png)

This should be changed to "level".

**Comment by aduth** [comment]
Closed by #4440

_(Aside: @rileybrook , as an FYI, you can use [keywords to automatically close issues by pull requests](https://help.github.com/articles/closing-issues-using-keywords/))_
